client/src/components/SortDirection/SortDirection.stories.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+
+import { primaryColor, secondaryColor } from "src/core/color_defs.js";
+
+import { SortDirection } from "./SortDirection.js";
+
+export default {
+  title: "SortDirection",
+  component: SortDirection,
+  parameters: {
+    backgrounds: {
+      default: "navy blue",
+      values: [
+        { name: "navy blue", value: primaryColor },
+        { name: "blue", value: secondaryColor },
+      ],
+    },
+  },
+};
+
+const Template = (args) => {
+  const [is_active1, set_active1] = useState(false);
+  const [is_active2, set_active2] = useState(false);
```

Comment:
Kinda minor but you can just use one state for this
```
const [is_ascending, set_is_ascending] = useState(false);
  return (
    <div
      onClick={() => set_is_ascending(!is_ascending)}
    >
      <SortDirection sortDirection="ASC" active={is_ascending} />
      <SortDirection sortDirection="DESC" active={!is_ascending} />
    </div>
  ); 
```

Reply:
I had the two states so both buttons would inactive initially

Reply:
okay, why should both buttons be inactive initially?

Reply:
both buttons are inactive initially on the tables on InfoBase so I thought I should do the same